src/templates/comment-reply-input.jsx

```diff
@@ -295,24 +295,23 @@ const CommentReplyInput = React.createClass({
                     context: 'verb: imperative',
                 });
                 submitLink = (
```

Comment:
I think we'll want to update these variable names as well, otherwise it may be a bit confusing.